auto_nag/scripts/crash_small_volume.py

```diff
@@ -176,13 +187,32 @@ def _has_severity_downgrade_comment(bug):
                 return True
         return False
 
+    def _is_severity_recently_changed_by_human_or_autonag(self, bug):
+        for entry in reversed(bug["history"]):
+            if entry["when"] < self.oldest_severity_change_date:
+                break
+
+            # We ignore bot changes except for autonag
+            if utils.is_bot_email(entry["who"]) and entry["who"] not in (
+                "autonag-nomail-bot@mozilla.tld",
+                "release-mgmt-account-bot@mozilla.tld",
+            ):
+                continue
+
+            for change in entry["changes"]:
+                if change["field_name"] == "severity":
```

Comment:
Nit: could rewrite this as: `if any(change["field_name"] == "severity" for change in entry["changes"])`